modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -58,6 +58,9 @@ hostname. If enabled, the route uses re-encrypt encryption. Defaults to `false`.
 |Array of additional routes to create. You provide the hostname and certificate
 for the route.
 
+|`rolloutStrategy`
+|Defines rollout strategy for the image registry deployment. Defaults to `RollingUpdate`
```

Comment:
Suggestion for consistency with the rest of the table: 
s/image registry/registry

And please add a period `.` after `RollingUpdate`.